### PR TITLE
Add RAKDAP1 bootloader upgrade section

### PR DESCRIPTION
**Pull request by mtuxpe**
Signed-off-by: msam-rak <71436549+msam-rak@users.noreply.github.com>

### CODE REVIEW

**Review by carlrakwireless — APPROVED** [review]
Hi @mtuxpe ,

What pyocd version did you do the guide? On our RAKDAP1 guide, it is still 0.27.3. Should we update it?

**Comment by mtuxpe** [comment]
pyocd version used was 0.30.3 but the last version is 0.32.3
pyOCD 0.32.3 released — pyOCD
<https://pyocd.io/posts/2021/12-30-pyocd-0.32.3-released.html>
I think we need to update the guide.

File: bootloader/RAK4630/README.md [hunk 1]
```diff
@@ -201,3 +202,42 @@ Steps as below:
...
+
+Install the RAK4631 module support package, using the following command:
+```c
+pyocd pack --install nrf52
+```
+### Connect RAKDAP1 adapter to SWD of the module
+
+Use the figure below as a guide to connect RAKDAP1 SWD to RAK4631.
+
+<img src="assets/rak4600-rakdap1.svg" width="80%" caption="RAKDAP1 connection" />
```
Comment (beegee-tokyo) [inline bootloader/RAK4630/README.md]: @mtuxpe 
Wrong image. 
You show a RAK4600, but this is for the RAK4631

**Review by beegee-tokyo — CHANGES_REQUESTED** [review]

File: bootloader/RAK4630/README.md (hunk 1 above)
Comment (mtuxpe) [reply bootloader/RAK4630/README.md]: @beegee-tokyo The new image shows only RAK4631and RAKDAP1 . I am unable to build a full image (RAK4631 + RAK4005-O)

**Comment by beegee-tokyo** [comment]
Good enough, thank you for the update @mtuxpe

**Pull request merged**